README.md

```diff
@@ -1,8 +1,7 @@
-# Project Name
+# Javascrip Chatbot
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
```

Comment:
Would be nice if you filled in more info about your approach to the problem, etc.

Reply:
Thank you for the review, I have now added more info about this project.